src/test/resources/features/complaints/ComplaintsTriage.feature

```diff
@@ -92,6 +92,17 @@ Feature: Complaints Triage
     And the summary should display the owning team as "Serious Misconduct"
     And the read-only Case Details accordion should contain all case information entered during the "Triage" stage
 
+  @ComplaintsRegression @ComplaintsWorkflow
+  Scenario: User completes the Case Triage stage for a BF case
+    Given I am logged into "CS" as user "BF_USER"
+    When I create a "BF" case and move it to the "Triage" stage
+    And I load and claim the current case
+    And I submit details on the Triage Capture Reason page
```

Comment:
Please use 'I get a "BF" case at the "Send" stage'. Then you wont require the second step about loading and claiming the case.